test/runtests.jl

```diff
@@ -1,24 +1,24 @@
 using AstroTime
+# TODO: Fix clashing `Period` exports
+import Dates
 using Test
-using ERFA
-using SPICE: furnsh, et2utc, utc2et
 
-furnsh(joinpath("data", "naif0012.tls"))
 AstroTime.load_test_eop()
 
 const speed_of_light = 299792458.0 # m/s
 const astronomical_unit = 149597870700.0 # m
 
-@timescale GMT UTC
-AstroTime.Epochs.getoffset(::GMTScale, ::CoordinatedUniversalTime, _, _) = 0.0
-AstroTime.Epochs.getoffset(::CoordinatedUniversalTime, ::GMTScale, _, _) = 0.0
+# TODO: Change me
```

Comment:
These examples are not great anymore and I will try to come up with new ones during the doc update.